sdk/storage/Azure.Storage.Files.DataLake/src/Sas/DataLakeSasBuilder.cs

```diff
@@ -90,6 +90,15 @@ public class DataLakeSasBuilder
         /// </summary>
         public string Path { get; set; }
 
+        /// <summary>
+        /// Defines whether or not the path is a directory. If set to true,
+        /// the path will be assumed to be a directory for a directory SAS.
+        /// If set to false or default, the path will be assumed as a path
```

Comment:
path will be assumed as a file?

Reply:
Can we add docs near the path setter about setting this as well?

Reply:
Good catch! haha "the path will be assumed as a path"